doc/source/api/index.rst

```diff
@@ -109,6 +109,7 @@ API
       .. toctree::
        :maxdepth: 1
 
+       Rust <https://github.com/georust/gdal>
```

Comment:
the list is apparently in alphabetical order, so this should be moved at the end after R

Reply:
@rouault Sorry I missed that. Thanks for taking care of it!